lib/reactotron-react-native/tsconfig.json

```diff
@@ -1,5 +1,4 @@
 {
-  "extends": "../../tsconfig.base.json",
```

Comment:
Why did this get removed? I know I had some issues with paths when we include this extends, but I thought this would be alright here

Reply:
That's a mistake, it must've happened when I was doing a bunch of debugging. I'll fix.